### PR TITLE
politis: pricelist management

### PR DESCRIPTION
**Pull request by p-nedelchev**
Price list management guide covers:
1. Create, edit and delete price lists. Discount management
2. Price lists comparison
3. Assign price list to customer and customer location. 
4. Special discounts. 
5. Discount priorities

### CODE REVIEW

**Comment by mgenov** [comment]
The screenshots need a little update as the customer name is misspelt: 'Тестови клеинт'

File: src/pages/politis/guide/pricelist-management/pricelist-management.bg.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+##### Отстъпки за дивизии
+* *Дивизия* - от падащ списък се избира дивизия, за която да се приложи избраната отстъпка. Това поле е *задължително*.
+* *Отстъпка* - отстъпка към дивизия в проценти. *Стойността трябва да е по-голяма от 0*. 
+
+След нанасяне на необходимите промени се натиска бутона *Запази*
+
+##### Отстъпки за варианти
+* *Име на варианта* - вариант, към който да се приложи избраната отстъпка. Предлага възможност за търсене на варианти. Това поле е *задължително*.
+* *Цена на дребно* - цена на дребно за варианта от продуктовия каталог.
+* *Отстъпка към дивизия* -
```
Comment (mgenov) [inline src/pages/politis/guide/pricelist-management/pricelist-management.bg.md]: Missing description.

File: src/pages/politis/guide/pricelist-management/pricelist-management.bg.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
```
Comment (mgenov) [inline src/pages/politis/guide/pricelist-management/pricelist-management.bg.md]: .

Comment (p-nedelchev) [reply src/pages/politis/guide/pricelist-management/pricelist-management.bg.md]: Description added.

**Review by mgenov — APPROVED** [review]
LGTM !

**Pull request merged**